locales/locales.php

```diff
@@ -1226,6 +1237,15 @@ public function __construct() {
 		$gn->lang_code_iso_639_2 = 'grn';
 		$gn->slug = 'gn';
 
+		$gn_py = new GP_Locale();
+		$gn_py->english_name = 'Guaraní (Paraguay)';
+		$gn_py->native_name = 'Avañe\'ẽ';
+		$gn_py->lang_code_iso_639_1 = 'gn';
+		$gn_py->lang_code_iso_639_2 = 'grn';
+		$gn_py->lang_code_iso_639_3 = 'grn';
+		$gn_py->country_code = 'py';
+		$gn_py->slug = 'gn-py';
```

Comment:
[Google](https://cloud.google.com/translate/docs/languages?hl=es-419#languages-pbmt) and [Facebook](https://www.facebook.com/settings?tab=language) support the Guaraní language, so I suggest you to add these lines:

```suggestion
		$gn_py->slug = 'gn-py';
		$gn_py->google_code = 'gn';
		$gn_py->facebook_locale = 'es_LA';
```